### DISCUSSION TITLE
Release API as a package?

### DISCUSSION THREAD

**Discussion by mward-sudo** [WebDevStudios/nextjs-wordpress-starter#372] [Ideas]
It seems you guys have done a lot of work on this project, and it appears to be the most complete Wordpress API implementation that I have seen in the node.js world.

Do you think the API you have created would be suitable to be released as a standalone NPM package that could be used in any node project? I don’t think that the code is particularly Next.js specific when it comes to the API.

I am thinking that there would be one package that contains all the core Wordpress API, and a series of packages that cover the API’s for various plugins.

I’m not in a position to do the work on this at the moment, but it is certainly something I would be willing to examine in a month or two when my availability is better.

Thoughts?

**Comment by gregrickaby** [comment]
@mward-sudo We have been discussing this internally. 

Our next focus will be to extract Algolia into its own NPM Package and then go from there.

**Reply by mward-sudo** [reply]
Excellent news
